### PR TITLE
ocitools-generate.1.md change specifiy to specifies

### PR DESCRIPTION
**Pull request by q384566678**
Signed-off-by: zhouhao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

**Comment by liangchenye** [comment]
LGTM

**Comment by wking** [comment]
8d3a3686 looks good to me.

**Comment by q384566678** [comment]
Ok, I'll change

File: man/ocitools-generate.1.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ compatible runtime like runC to run a container.
   /HOST-DIR:/CONTAINER-DIR`, runc bind mounts `/HOST-DIR` in the host
   to `/CONTAINER-DIR` in the OCI container. The `OPTIONS` are a comma
   delimited list and can be: [rw|ro] The `HOST_DIR` and
-  `CONTAINER-DIR` must be absolute paths such as `/src/docs`.  You
+  `CONTAINER-DIR` must be an absolute paths such as `/src/docs`.  You
   can add `:ro` or `:rw` suffix to a volume to mount it read-only or
   read-write mode, respectively. By default, the volumes are mounted
```
Comment (Mashimiao) [inline man/ocitools-generate.1.md]: This is not a right fix. You should refer  to the context.

Comment (q384566678) [inline man/ocitools-generate.1.md]: Well, I was wrong

**Comment by Mashimiao** [comment]
[1d90078](https://github.com/opencontainers/ocitools/pull/125/commits/1d900789dd9b21cc780d08710c87f2e5e7e0a51c) looks good to  me.

**Comment by liangchenye** [comment]
LGTM

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**